cmd/weaver-gke/deploy.go

```diff
@@ -46,3 +53,21 @@ var deploySpec = tool.DeploySpec{
 		return gke.LogSource(config)
 	},
 }
+
+// getToolVersion returns the version of the tool binary.
+func getToolVersion() (string, error) {
+	info, ok := debug.ReadBuildInfo()
+	if !ok {
+		// Should never happen.
+		return "", fmt.Errorf("tool binary must be built from a module")
+	}
+	version := info.Main.Version
+	const develToolVersion = "(devel)"
+	if version == develToolVersion {
+		// Locally-compiled tool binary. Return a version that's guaranteed to
+		// be lexicographically greater than any previously returned version
+		// (e.g., v0.10.0).
+		return fmt.Sprintf("vdevel%d", time.Now().Unix()), nil
```

Comment:
I'm probably misunderstanding something, but because every `vdevel...` version is lexicographically grater than every `v0.X.X` version, does this mean that once a devel version is used, a `v0.X.X` will never be used?

Reply:
Yes, that is true. This is perhaps okay, as someone who uses a developer version should be savvy enough to stop the nanny before using a prod version.

Reply:
Got it!